onnxruntime/core/mlas/lib/amx_common.h

```diff
@@ -0,0 +1,95 @@
+/*++
```

Comment:
is this used only in one file? if so can we just put this in that file?

Reply:
We plan to use this header for future AMX kernels like convolutions. Therefore, these functions are kept in a separate file.